app/guid-file/route.ts

```diff
@@ -54,11 +62,92 @@ export default class GuidFile extends Route {
                 this.intl.t('general.presented_by_osf'),
             language: this.metadata.language ? this.metadata.language : undefined,
         };
-        this.set('headTags', this.metaTags.getHeadTags(metaTagsData));
+
+        // Google Structured Data
+        const { environment } = config;
+        const parentGuid = Object.values(model.links['relationships']['target']['data']);
+        const parentGuidString: string = parentGuid[1] as string;
+        const jsonLD: Promise<{}> = await this.returnStructuredData(parentGuidString, environment);
+
+        if (jsonLD) {
+            this.set('structuredData', jsonLD);
+        }
+
+        const jsonString: string = this.structuredData ?
+            JSON.stringify(this.structuredData) : JSON.stringify({ isAccessibleForFree : true });
+
+        const scriptTagsData = {
+            type: 'application/ld+json',
+            src: `osf.io/${parentGuidString}/metadata/?format=google-dataset-json-ld`,
```

Comment:
QA and Accessibility protocols request it. Google's sample code is boilerplate and should be extended where necessary and feasible.

Reply:
The `src` attribute is a link that points to an external script which the browser should make additional `GET` requests to retrieve the content, while the content included between the open and closing `<script>` tags does not need additional `GET`.

I don't know how the browser would handle the case where both are present. Should be browser append the two JS scripts? Should one take precedence over another? If both are identical, should they be executed twice? There are so many questions. And I guess the actual behavior is browser-specific.

I am not familiar with any QA/A11y protocols that have such requirements. We definitely have lots of places in the code base where we have either the `src` attribute or the content, but not both. I can't think of any use case that requires both. If you do know of any please let me know and we'll take a look.

Meanwhile, please remove the `src` attribute.

Reply:
This is now removed, but have still included it for our discussion today. It is an alternative to using it in the body. In this situation using Google's format should be correct. Our team had mentioned that the src link should download so I included it but believe the src attribute, not the object between the script tags, will result in that being the default. If that happens, it could lead to users having downloads for a json file and have no idea where it came from. So yes, this is now removed.